### ISSUE TITLE
FPCUpDeluxe failing to install from scratch

### ISSUE THREAD

**Issue by SuperDisk** [LongDirtyAnimAlf/fpcupdeluxe#224]
I'm trying to install any version of FPC + Lazarus. Every time, I'm running into a strange error that doesn't seem to provide any details. Here's the full log, with "extra verbosity" turned on:

https://gist.github.com/SuperDisk/da8efeb8b56d47609668c55295adddfa

One thing I noticed is that FPCUD seems to try to run `doskey` but fails. Not sure if it's related.

Running on Windows 10 x64, FPCUPdeluxe V1.6.8p for x86_64-win64-win32

**Comment by SuperDisk** [comment]
Aha-- I believe I found the problem. I had created an `autorun` command script which would run every time I started the Windows command processor, to bind a few aliases and set some variables that I could use. It seems like FPCUPDeluxe is spawning some CMD processes, and my autorun script is somehow interfering with it-- that's where the `doskey` error comes from.

https://www.thekidds.org/blog/2012/07/06/rc-file-for-your-windows-command-prompt/

I guess this problem is solved. If anyone wants to know the answer to this problem, I got rid of the autorun script and FPCUD seems to be working correctly now.
